README.md

```diff
@@ -35,6 +35,12 @@ Image Reading, Metadata Conversion, and Image Writing for Microscopy Images in P
 **Stable Release:** `pip install aicsimageio`<br>
```

Comment:
Does `pip install aicsimageio` (no module specified) default to `[all]` or '[base-imageio]' or ?

Reply:
Defaults to no extras. So just TIFF + OME-TIFF would be supported.

`pip install aicsimageio` == TIFF + OME-TIFF
`pip install aicsimageio[czi]` == TIFF + OME-TIFF + CZI
`pip install aicsimageio[base-imageio]` == TIFF + OME-TIFF + all the imageio ones
`pip install aicsimageio[czi,base-imageio]` == TIFF + OME-TIFF + CZI + all the imageio ones
`pip install aicsimageio[all]` == all formats

Reply:
Perfect.  Thanks!  Looks good